index.bs

```diff
@@ -3184,42 +3164,38 @@ The [=remote end steps=] with |command parameters| are:
 
   <!-- This is based on step 5 of https://w3c.github.io/webdriver/#new-window,
        but without using the "current browsing context" concept. -->
-  1. Create a new [=top-level browsing context=] by running
-     the [=/create a new top-level traversable=] steps with null and empty string,
+  1. Let |navigable| be the result of trying to [=/create a new top-level traversable=] steps with null and empty string,
      and setting the [=associated user context=] for the newly created [=/top-level traversable=] to |user context|.
-     Which OS window the new [=/browsing context=] is created in depends on |type| and |reference context|:
+     Which OS window the new [=/navigable=] is created in depends on |type| and |reference navigable|:
```

Comment:
Should we use top-level traversable throughout this step and set of conditions?

Reply:
Updated